packages/graphql-adapter/src/adapter/adapter.ts

```diff
@@ -149,12 +115,21 @@ class GraphQlAdapter implements TGraphQlAdapterInterface {
 
         if (this.#didFlagsChange(nextFlags)) {
           if (adapterArgs.cacheIdentifier) {
-            const cache = await this.#getCache(adapterArgs.cacheIdentifier);
+            const cache = await getCache(
+              adapterArgs.cacheIdentifier,
+              adapterIdentifiers.graphql,
+              this.#adapterState.user?.key
+            );
 
             cache.set(nextFlags);
           }
 
           this.#adapterState.flags = nextFlags;
+
+          if (adapterArgs.cacheMode === 'lazy') {
```

Comment:
In lazy mode we just write to the cache. No flushing of updates. The next configuration or reconfiguration (can be a reload) will flush this state "into React".